app/src/main/java/com/venuenext/venuenextsdkdemo/MainActivity.kt

```diff
@@ -18,6 +18,7 @@ import com.venuenext.vncore.protocol.TicketingInterface
 import com.venuenext.vncore.protocol.WalletInterface
 import com.venuenext.vncoreui.LifecycleCoroutineScope
 import com.venuenext.vnlocalytics.localytics.LocalyticsAnalytics
+import com.venuenext.vnorder.VenueNextOrders
```

Comment:
This is accidental? 👀

Reply:
Yeah. That was from using the wrong method to configure payment processable. I’ll delete that import.